source/documentation/retire-packages.html.md

```diff
@@ -0,0 +1,50 @@
+---
+author: ykarel
+title: Retiring a package
+---
+
+# Retiring a package from RDO
+
+1. toc
+{:toc}
+
+### How to remove an OpenStack package from RDO Trunk
+
+Package for a project can exist across different releases. So retiring it in RDO needs to go via stages,
+To remove a package, following steps are required:-
+
+Stage 1:- Project Source is retired/deleted upstream, but it's package still need to be maintained because it's required by other projects
+
+- Send a review to `rdoinfo` like [Example review](https://review.rdoproject.org/r/#/c/20387/) to pin(add source-branch: `<good commit>`) the project against the tag or commit, so package get's build from pinned commit rather than latest commit(deleted source code).
```

Comment:
This example is confusing because it's pinning multiple releases while the regular case should be that only master tags should be pinned (in oslo-sphinx they didn't create stable/rocky nor stable/stein). I guess we don't have a better one?

Reply:
yes right didn't find a better one for retiring case.